plugins/generic/metadataExport/MetadataExportBlockPlugin.inc.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file plugins/generic/metadataExport/MetadataExportBlockPlugin.inc.php
+ *
+ * Copyright (c) 2013-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class MetadataExportBlockPlugin
+ * @ingroup plugins_generic_metadataExport
+ *
+ * @brief Class for block component of metadata export plugin
+ */
+
+import('lib.pkp.classes.plugins.BlockPlugin');
+
+class MetadataExportBlockPlugin extends BlockPlugin {
+	
+	/* @var parentPluginName String - name of the parent plugin object */
```

Comment:
For consistency (we're not perfect in this regard), `@var` declarations should be...

    /** @var string Name of the parent plugin object */
    var $parentPluginName;

Note that the variable name is only specified once.